podcasts/ArrayExtension.swift

```diff
@@ -4,4 +4,11 @@ extension Array {
             Array(self[$0 ..< Swift.min($0 + size, count)])
         }
     }
+
+    @discardableResult
+    mutating func insert(_ element: Element, safelyAt at: Int) -> Int {
+        let indexToInsert = at <= count ? at : count
```

Comment:
Should we just use `.min` instead of manually doing the check?

```suggestion
        let indexToInsert = Swift.min(at, count)
```